### DISCUSSION TITLE
Is datapane compatible with ipywidgets?

### DISCUSSION THREAD

**Discussion by MemphisMeng** [datapane/datapane#51] [Ideas]
I want to work around one of Jupyter Notebook's shortages that is its incompatibility with objects from ipywidgets. Can it be done with datapane?

**Comment by lanthias** [comment]
Hey - thanks a lot for the comment. Datapane is not compatible, as ipywidgets require a running Python server, and Datapane reports are pure HTML and don't require Python.

That said, I would love to learn a bit about your use-case and see if I can point you in the direction of another way to solve the problem outside of using ipywidgets.

**Reply by MemphisMeng** [reply]
Thanks, I just found that ipywidgets is totally not allowed in all kinds of static showcases.
